spec/spec_helper.rb

```diff
@@ -2,8 +2,9 @@
 
 # Notes
 # We need to load vairables that 'beef' usually does for us
-## config
+## config (this isn't used or is shadowed by the monkey patching, needs a further look to fix properly)
 config = BeEF::Core::Configuration.new('config.yaml')
+## I haven't been able to work out why these are global variables, best practive says it needs to be changed but its used in beef.
```

Comment:
```suggestion
config = BeEF::Core::Configuration.new('config.yaml')
## I haven't been able to work out why these are global variables, best practice says it needs to be changed but its used in beef.
```

I think these were used in BeEF to find the user's home directory for storing stuff like loot and logs. They're global because they're not defined in the config file, and BeEF objects are instantiated only with the config. Fixing this should be done as part of reworking the configuration loading.